### PR TITLE
update actions to push new release to krew-index

### PR DESCRIPTION
**Pull request by rajatjindal**
Update release yml to also update krew-index repo.

Also request you to please disable krew-release-bot github app to avoid sending duplicate requests.

### CODE REVIEW

**Comment by ahmetb** [comment]
This is dope. 
Do you want to delete the Github App altogether?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -20,3 +20,7 @@ jobs:
         args: release --rm-dist
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Update new version in krew-index
+      uses: rajatjindal/krew-release-bot@v0.0.20
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ahmetb) [inline .github/workflows/release.yml]: Do we need gh token here?
I think many people will be against sending their token to an image (or a backend) we own.

Comment (rajatjindal) [reply .github/workflows/release.yml]: the token is being used to verify authorization of submitting the request to release the plugin. 

we can remove the verification part if you think it won't be a security concern.

Thanks
Rajat Jindal

**Comment by rajatjindal** [comment]
> This is dope.
> Do you want to delete the Github App altogether for everyone? That might be better.

the bot now supports two endpoints, one for webhook and one for action. 

i would think we can continue to have both and let user make a choice? personally I also liked goreleaser way of doing the release (though I am struggling to get this to work because i do not use go modules yet).

Thanks
Rajat Jindal

File: .github/workflows/release.yml (hunk 1 above)
Comment (ahmetb) [reply .github/workflows/release.yml]: I don’t think it’s a concern.

What’s the worst PR that can come to krew-index with this? I can’t imagine anything bad.

Comment (ahmetb) [reply .github/workflows/release.yml]: Lets remove this.

**Comment by ahmetb** [comment]
I think we would advertise the GitHub action. Because it can print output/progress etc. GH App wasn’t usable at all for me and for many others.

File: .github/workflows/release.yml (hunk 1 above)
Comment (rajatjindal) [reply .github/workflows/release.yml]: done.

**Comment by rajatjindal** [comment]
> I think we would advertise the GitHub action. Because it can print output/progress etc. GH App wasn’t usable at all for me and likely for many others.

sure sounds good to me. I have remove the token requirement from github action. 

Thanks
Rajat Jindal

**Comment by ahmetb** [comment]
This worked like a champ
image

cc: @corneliusweig

**Comment by corneliusweig** [comment]
Wow, this is nice!

**Pull request merged**